src/version.js

```diff
@@ -1,6 +1,6 @@
 const Product = {
   name: 'oVirt Basic Portal',
-  version: '1.1.0',
```

Comment:
This file does not need to be modified.

It is regenerated by ./autogen.sh according to configure.ac.